R/pal-dichromat.r

```diff
@@ -1,5 +1,6 @@
 #' Dichromat (colour-blind) palette (discrete).
 #'
+#' @noMd
```

Comment:
What do you need this for?

Reply:
without it the \Sexpr that follows breaks - seems to be the same bug documented 
 in klutometis/roxygen#682.

Reply:
Bummer